components/ColorPickerModal.js

```diff
@@ -92,6 +98,7 @@ export default function ColorPickerModal({ initialColor, onColorSelected, onClos
       </View>
       <ScrollView
         contentContainerStyle={styles.scrollViewContent}
+        scrollEnabled={activeTab !== 'wheel'}
```

Comment:
Why are we disabling scroll for this? can you add a small comment as this is not obvious by looking at the code

Reply:
So, I am disabling scroll for the color picker because when you drag the wheel the view moves.
So based on this [issue](https://github.com/Naeemur/react-native-wheel-color-picker/issues/4) I disabled the scroll for this specific tab